package.json

```diff
@@ -26,7 +26,7 @@
   "engines": {
     "node": "18.x",
     "pnpm": ">=8.6.0",
-    "vscode": ">=1.46.0"
+    "vscode": ">=1.47.0"
```

Comment:
Bumping the minimum VS Code version by one minor version, to 1.47. This is required to be able to pass a title to the save dialog.